docs/manual/index-checker.tex

```diff
@@ -156,12 +156,16 @@ \section{Lower bounds\label{index-lowerbound}}
 
 The Index Checker issues an error when
 a sequence is indexed by an integer that might be negative.
-It uses a type system (Figure~\ref{fig-index-int-types}) with the following
+The Lower Bound Checker uses a type system (Figure~\ref{fig-index-int-types}) with the following
 qualifiers:
 
 \begin{description}
 \item[\refqualclass{checker/index/qual}{Positive}]
   The value is 1 or greater, so it is not too low to be used as an index.
+  Note that this annotation is trusted by the Constant Value Checker,
```

Comment:
This detail seems distracting at this point in the manual; mentions elsewhere seem adequate.
How about making the Index Checker issue a warning if it ever encounters a programmer-written `@Positive` annotation and the Lower Bound Checker is not being run?  That seems like a less error-prone solution, and another benefit is that it enables all the relevant warnings to be removed from the manual.